modules/gdscript/editor/gdscript_highlighter.cpp

```diff
@@ -395,13 +397,41 @@ Dictionary GDScriptSyntaxHighlighter::_get_line_syntax_highlighting_impl(int p_l
 			in_member_variable = false;
 		}
 
-		if (!in_node_path && in_region == -1 && (str[j] == '^')) {
+		if (j > 0 && (str[j] == '&' || str[j] == '^' || str[j] == '%' || str[j] == '+' || str[j] == '-' || str[j] == '~')) {
+			int k = j - 1;
+			while (k > 0 && (str[k] == '\t' || str[k] == ' ')) {
```

Comment:
Can use the `is_whitespace` macro instead, not sure if it's worth cleaning up the other places.

```suggestion
			while (k > 0 && is_whitespace(str[k])) {
```

Reply:
I'll clean up other places too.